src/main/java/com/salesforce/storm/spout/sideline/handler/SidelineSpoutHandler.java

```diff
@@ -90,7 +90,7 @@
     /**
      * Timer for periodically checking the state of the VirtualSpouts being managed.
      */
-    private final Timer timer = new Timer();
```

Comment:
We could leave this final, tho I kind of like the idea of naming the threads with some contextual info.